Doctrine/Phpcr/StaticContent.php

```diff
@@ -11,14 +11,13 @@
 
 namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr;
 
-use Doctrine\ODM\PHPCR\HierarchyInterface;
 use PHPCR\NodeInterface;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent as ModelStaticContent;
 
 /**
  * PHPCR specific static content.
  */
-class StaticContent extends ModelStaticContent implements HierarchyInterface
```

Comment:
i created a 2.0 branch for the odm, see
https://github.com/doctrine/phpcr-odm/pull/711

if i understand composer correctly, you can simply depend on phpcr-odm
`2.0@dev` to make this work. though there are likely conflicts then with
other dependencies?

i'd like to keep this as a 2.0 branch for now, until we get closer to
release the cmf 2.0 versions and then switch phpcr-odm over to 2.0